opencti-platform/opencti-front/src/private/components/settings/users/UserHistory.tsx

```diff
@@ -65,6 +66,7 @@ const UserHistory: FunctionComponent<UserHistoryProps> = ({
         ],
         operator: 'eq',
         mode: 'or',
+        id: generateUUID(), // because filters in the URL
```

Comment:
why do we need an `id` in filters ? and it's using a method from `tests_e2e`, I don't think its the right import.

Reply:
in filtersUtils.tsx, I see that you use : 
`id: uuid(),`
I guess if we have now a logic with id in filters, we should use this same method `uuid()`, no?

Reply:
I change by uuid().